expression/builtin.go

```diff
@@ -136,6 +136,36 @@ func newBaseBuiltinFunc(ctx sessionctx.Context, funcName string, args []Expressi
 	return bf, nil
 }
 
+func newFieldTypeForBaseBuiltinFunc(funcName string, retType types.EvalType, ec *ExprCollation) *types.FieldType {
```

Comment:
maybe rename to `newReturnFieldTypeForBaseBuiltinFunc`?

Reply:
ok, updated.